man/systemd.exec.xml

```diff
@@ -1457,6 +1451,29 @@
       </varlistentry>
 
       <varlistentry>
+        <term><varname>RestrictNamespaces=</varname></term>
+
+        <listitem><para>Restricts access to Linux namespace functionality for the processes of this unit. For details
+        about Linux namespaces, see
+        <citerefentry><refentrytitle>namespaces</refentrytitle><manvolnum>7</manvolnum></citerefentry>. Either takes a
+        boolean argument, or a space-separated list of namespace type identifiers. If false (the default), no
+        restrictions on namespaces are made. If true, access to any kind of namespacing is prohibited. If neither, a
+        space-separated list of namespace type identifiers must be specified, consisting of any combination of:
+        <constant>cgroup</constant>, <constant>ipc</constant>, <constant>net</constant>, <constant>mnt</constant>,
+        <constant>pid</constant>, <constant>user</constant> and <constant>uts</constant>. Any namespace type listed is
```

Comment:
Here too, what we are actually restricting, is the _creation_ or switching of namespaces, not access to them.